src/main/resources/db/migration/V1.13.0__Add_gacha_type_colmns.sql

```diff
@@ -0,0 +1,3 @@
+USE seichiassist;
+
+ALTER TABLE gachadata ADD event_name VARCHAR(7) DEFAULT NULL;
```

Comment:
マイグレーションはできるだけ冪等にしてください
```suggestion
ALTER TABLE IF EXISTS gachadata ADD event_name IF NOT EXISTS VARCHAR(7) DEFAULT NULL;
```